deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -304,75 +304,124 @@ spec:
+    control plane and you will not see the desired outcome. For more information, use this [link](https://github.com/openshift/federation-dev/blob/master/docs/kubefed-scope.md) 
 
-    **Important Note**: The instructions in this guide refer to the namespace
-    where federation is installed as `<namespace>` in command examples. The
-    upstream user guide uses `kube-federation-system` as the name of this namespace.
+    * To add federation capability to your namespace, you are only required to install kubefed in the hosting cluster; 
+    there is no need to install kubefed in the target clusters (and no effect gained from doing so).
 
-    ### Get the kubefedctl CLI tool
+    * The instructions in this guide refer to the namespace where `KubeFed` is installed as `<namespace>` in command examples. 
+    The upstream user guide uses `kube-federation-system` as the name of this namespace.
 
-    Before you use kubefed, you should ensure that you're using the binary
-    for this version. You can get it from the releases on GitHub.
+    ### Download binaries
 
-        curl -Ls https://github.com/kubernetes-sigs/kubefed/releases/download/v0.1.0-rc3/kubefedctl-0.1.0-rc3-linux-amd64.tgz | tar xz
+    Once the `kubefed` control plane is up and running, a user can download the `kubefedctl` command-line tool to communicate with it. 
+    You can get the latest release on [GitHub](https://github.com/kubernetes-sigs/kubefed/releases).
 
+    ```
+     $ curl -Ls https://github.com/kubernetes-sigs/kubefed/releases/download/v0.1.0-rc4/kubefedctl-0.1.0-rc4-linux-amd64.tgz
+     $ tar -xvzf kubefedctl-0.1.0-rc4-linux-amd64.tgz
+     $ chmod u+x kubefedctl
+     $ sudo mv kubefedctl /usr/local/bin/ # make sure the location is in the PATH
+    ```
     ### Joining Clusters
 
-    Use the `kubefedctl join` command to connect clusters you want federation to
-    spread resources to. `kubefedctl join` reads information about how to connect
-    to the joining cluster and the cluster hosting federation from your
-    KUBECONFIG.
-
-        kubefedctl join cluster-name \
-          --cluster-context mycluster \            # name of a KUBECONFIG context for the cluster to join
-          --host-cluster-context mycluster \       # name of a KUBECONFIG context for the hosting cluster
-          --kubefed-namespace=<namespace> \        # namespace where federation is deployed
-          -v 2
-     
-     Note: if you're using an openshift cluster for deploying the kubefed-operator then
+    Use the `kubefedctl join` command to connect clusters you want federation to spread resources to. The `join` operation reads information 
+    about how to connect to the joining cluster and the cluster hosting federation from your KUBECONFIG.
+
+    ```
+    $ kubefedctl join cluster-name \
+        --cluster-context mycluster \            # name of a KUBECONFIG context for the cluster to join
```

Comment:
Maybe note that `--cluster-context` is only required if the context name differs from the cluster name?